Snake/BusinessLayer/MazeLevel.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BusinessLayer
+{
+    public enum MazeLevel
+    {
+        Easy = 1,
```

Comment:
@wjoneil @shreyasn995  If not given a value, it will starts from 0. As long as the first one is given a value. the following element will increase the value by 1 automatically, i think?